library/pod-security-policy/host-filesystem/src.rego

```diff
@@ -2,15 +2,19 @@ package k8spsphostfilesystem
 
 violation[{"msg": msg, "details": {}}] {
     volume := input_hostpath_volumes[_]
-    not input_hostpath_allowed(volume)
+    input_hostpath_violation(volume)
     msg := sprintf("HostPath volume %v is not allowed, pod: %v. Allowed path: %v", [volume, input.review.object.metadata.name, input.parameters.allowedHostPaths])
 }
 
-input_hostpath_allowed(volume) {
-    # An empty list means there is no restriction on host paths used
+input_hostpath_violation(volume) {
```

Comment:
What do we want to do if an admin doesn't provide the parameter at all? Should that be treated the same way as setting allowed hostpaths to empty?

Reply:
Per Kubernetes PSP [doc](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems):
> An empty list means there is no restriction on host paths used. 

In the pod security policy standard [doc](https://kubernetes.io/docs/concepts/security/pod-security-standards/):
> HostPath volumes must be forbidden.

Otherwise, the values provided will be whitelisted. 

In the new standard doc, it does not explicitly call out what is the default behavior for empty list. Should this be consistent with existing k8s PSPs or should empty list mean restrict all? 

cc @tallclair

Reply:
_bump_ @tallclair any thoughts on this?